src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+body {
+  display: flex;
+  justify-content: center;
+  font-family: "Roboto", sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+.product {
+  width: 200px;
+  height: 408px;
```

Comment:
```suggestion

```
It's a bad practice to hardcode the height of the block, it should take the height of its content

Reply:
Not fixed from the previous review